tests/acceptance/standard/test_hooks_processor.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from tests.acceptance import run_gitlabform, create_group, create_project
```

Comment:
Don't need to import `create_group` and `create_project` here, right? They seem to be unused in this test file.

Reply:
oops... again!